client/my-sites/site-settings/form-general.jsx

```diff
@@ -439,7 +447,21 @@ class SiteSettingsFormGeneral extends Component {
 				/>
 
 				<FormSettingExplanation>
-					{ translate( 'Choose a city in your timezone.' ) }
+					{ translate( 'Choose a city in your timezone.' ) }{' '}
+					{ translate( 'We think you might be in {{timeZoneLink/}} (click to select it).', {
+						components: {
+							timeZoneLink: (
+								<Button
+									onClick={ setGuessedTimezone }
+									borderless={ true }
+									compact={ true }
+									className="site-settings__general-settings-set-guessed-timezone"
+								>
+									{ guessedTimezone }
+								</Button>
```

Comment:
Is using a `Button` and styling it like a link a standard practice here?  It looks fine and works fine, just wondering because my first thought would have been to use an `a`.

Reply:
To be honest: I don't know. `eslint` has called me out for using a `<a href="">` "as button," so I assumed this was the expected way to deal with it.